### PR TITLE
refactor: [M3-6522] - Toggle component v7 story migration and cleanup

### PR DESCRIPTION
**Pull request by coliu-akamai**
## Description 📝
Small PR to migrate Toggle component story to a V7 (.tsx) story!

## Changes  🔄
List any change relevant to the reviewer.
- New v7 .tsx story to replace the .mdx one
- Import/export cleanup + eslint fixes on files I visited

### Prerequisites
- pull code locally
- run yarn && yarn storybook

### Verification steps 
- navigate to http://localhost:6006/?path=/docs/components-toggle--documentation + confirm story loads properly
- Confirm no regression for toggle on CM

## As an Author I have considered 🤔

*Check all that apply*

- [ ] 👀 Doing a self review
- [ ] ❔ Our [contribution guidelines](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md)
- [ ] 🤏 Splitting feature into small PRs
- [x] ➕ Adding a changeset
- [x] 🧪 Providing/Improving test coverage
- [ ] 🔐 Removing all sensitive information from the code and PR description
- [ ] 🚩 Using a feature flag to protect the release
- [ ] 👣 Providing comprehensive reproduction steps
- [ ] 📑 Providing or updating our documentation
- [ ] 🕛 Scheduling a pair reviewing session
- [ ] 📱 Providing mobile support
- [ ] ♿  Providing accessibility support

### CODE REVIEW

File: packages/manager/src/components/Toggle/Toggle.tsx
Comment (bnussman-akamai) [inline packages/manager/src/components/Toggle/Toggle.tsx:1]: We could move `packages/manager/src/components/Toggle/Toggle.tsx` to `packages/manager/src/components/Toggle.tsx` to simplify things a bit, but this is optional.

Comment (coliu-akamai) [reply packages/manager/src/components/Toggle/Toggle.tsx:1]: 👀 In general is there a rule of thumb for the min. amount of files that should be in a package for organization? There're currently three toggle related files - the tests, story, and toggle component itself - in the package. Would it be too much clutter if I got rid of the `Toggle` package?

Comment (bnussman-akamai) [reply packages/manager/src/components/Toggle/Toggle.tsx:1]: I personally don't mind if the `src/components` folder is massive but it's definitely fair to keep the `src/components/Toggle` folder around if there is a few files

Comment (abailly-akamai) [reply packages/manager/src/components/Toggle/Toggle.tsx:1]: I like it in its own directory - makes it easier at a glance to see how things are organized and if for example a component is missing a test, a story etc. It's a bit more portable too, even if that does not matter now. Has the drawback of the import being longer but not a big deal.

File: packages/manager/src/components/Toggle/Toggle.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+/**
+ * Example of a Toggle with an interactable tooltip
+ */
+export const InteractableTooltipToggle: StoryObj<ToggleProps> = {
+  render: () => <Toggle interactive={true} tooltipText={EXAMPLE_TEXT} />,
+};
+
+const meta: Meta<ToggleProps> = {
+  component: Toggle,
+  title: 'Components/Toggle',
```
Comment (abailly-akamai) [inline packages/manager/src/components/Toggle/Toggle.stories.tsx:30]: define your args with defaults here so we can interact with the component directly in the storybook docs. doesn't have to be all the props, but at least the ones we'd one to play with when looking at the component

**Review by abailly-akamai — APPROVED** [review]
Looks good! Approving pending adding args to the story

- story and self documentation looks great ✅ 
- tests ✅ 
- no visual regression or missing import ✅

**Review by bnussman-akamai — APPROVED** [review]

**Pull request merged**